addons/html_builder/static/src/sidebar/invisible_elements_panel.js

```diff
@@ -55,25 +55,29 @@ export class InvisibleElementsPanel extends Component {
         //     └ descendantInvisibleSnippet
         //          └ descendantOfDescendantInvisibleSnippet
         //               └ etc...
-        const createInvisibleEntries = (snippetEls, isDescendant) =>
+        const createInvisibleEntries = (snippetEls, parentEl = null) =>
             snippetEls.map((snippetEl) => {
                 const descendantSnippetEls = descendantPerSnippet.get(snippetEl);
                 // An element is considered as "RootParent" if it has one or
                 // more invisible descendants but is not a descendant.
                 const invisibleElement = {
                     snippetEl: snippetEl,
                     name: getSnippetName(snippetEl),
-                    isRootParent: !isDescendant && !!descendantSnippetEls,
-                    isDescendant,
+                    isRootParent: !parentEl && !!descendantSnippetEls,
+                    isDescendant: !!parentEl,
                     isVisible: snippetEl.dataset.invisible !== "1",
                     children: [],
+                    parent: parentEl,
```

Comment:
Let's maybe call it `parentEl` instead of `parent` so that we are sure that we speak of an html element just by reading the variable :+1: